auslib/web/balrog_api.py

```diff
@@ -0,0 +1,76 @@
+import connexion
+from connexion.operation import Operation
+from connexion.resolver import Resolver
+
+
+RANDOM_DIGITS = 6
+
+
+class BalrogApi(connexion.Api):
```

Comment:
OK, I'll made changes this weekend.
I sent an e-mail to connexion devs asking if they plan expose this feature. Waiting answers haha.